src/elements/content-sidebar/SidebarNavSign.tsx

```diff
@@ -22,10 +24,24 @@ export type Props = PlainButtonProps & {
 
 export const PlaceholderTooltip = ({ children }: { children: React.ReactNode }) => children;
 
-export function SidebarNavSign({ intl, status, targetingApi, ...rest }: Props) {
+export function SidebarNavSign({ blockedReason, intl, isDisabled, status, targetingApi, ...rest }: Props) {
     const isTargeted = targetingApi && targetingApi.canShow;
     const FtuxTooltip = isTargeted ? TargetedClickThroughGuideTooltip : PlaceholderTooltip;
     const label = intl.formatMessage(status === 'active' ? messages.boxSignSignature : messages.boxSignRequest);
+    let tooltipMessage = label;
+
+    switch (blockedReason) {
```

Comment:
Is this logic fine here or should this get moved one project up?

Reply:
I think it should be okay here.